### PR TITLE
Docs for `14.0.0` upgrade

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description

Docs to accompany #1112 

- No more `Vector3f` = adjust tutorial contents. 
- No more `AlphaLocator` stuff.

#### Tests

- [x] Test tutorial code

### CODE REVIEW

File: docs/projects/fps/tutorial.md [hunk 1]
```diff
@@ -113,13 +113,13 @@ The `HealthPickup` entity is a new type of entity, so we must create a new entit
...
 ```csharp
-public static EntityTemplate HealthPickup(Vector3f position, uint healthValue)
+public static EntityTemplate HealthPickup(Vector3 position, uint healthValue)
 {
     // Create a HealthPickup component snapshot which is initially active and grants "heathValue" on pickup.
     var healthPickupComponent = new Pickups.HealthPickup.Snapshot(true, healthValue);
 
     var entityTemplate = new EntityTemplate();
-    entityTemplate.AddComponent(new Position.Snapshot(new Coordinates(position.X, position.Y, position.Z)), WorkerUtils.UnityGameLogic);
+    entityTemplate.AddComponent(new Position.Snapshot(Coordinates.FromUnityVector(position), WorkerUtils.UnityGameLogic);
```
Comment (paulbalaji) [inline docs/projects/fps/tutorial.md]: does position.ToCoordinates() not work?

File: docs/reference/workflows/monobehaviour/worker-connectors.md [hunk 2]
```diff
@@ -110,14 +107,14 @@ Each of the [connection flows]({{urlRoot}}/reference/concepts/connection-flows)
...
 
-- `CommandLineConnectionFlowInitializer` which can initialize the `ReceptionistFlow,` `LocatorFlow`, and `AlphaLocatorFlow` from command line parameters.
-- `MobileConnectionFlowInitializer` which can initialize the `ReceptionistFlow` and `AlphaLocatorFlow`.
+- `CommandLineConnectionFlowInitializer` which can initialize the `ReceptionistFlow` and `LocatorFlow` from command line parameters.
+- `MobileConnectionFlowInitializer` which can initialize the `ReceptionistFlow` and `LocatorFlow`.
 
 ## How do I customize the connection flows
 
-The Locator and AlphaLocator connection flows both have callbacks to the user during the connection process. The `LocatorFlow` and `AlphaLocatorFlow` objects provide a reasonable default implementation for these callbacks, but you may want to customize this to suit your connection & authentication logic. 
+The Locator connection flow has callbacks to the user during the connection process. The `LocatorFlow` object provide a reasonable default implementation for these callbacks, but you may want to customize this to suit your connection & authentication logic.
```
Comment (paulbalaji) [inline docs/reference/workflows/monobehaviour/worker-connectors.md]:
```suggestion
The Locator connection flow has callbacks to the user during the connection process. The `LocatorFlow` object provides a reasonable default implementation for these callbacks, but you may want to customize this to suit your connection & authentication logic. 
```

File: docs/projects/fps/tutorial.md (hunk 1 above)
Comment (jamiebrynes7) [reply docs/projects/fps/tutorial.md]: Unsure which imports are required for that, so I went with this 😬

File: docs/variable.properties [hunk 3]
```diff
@@ -1,2 +1,2 @@
 current_version=0.2.6
-worker_sdk_version=13.8
\ No newline at end of file
+worker_sdk_version=14.0
```
Comment (zeroZshadow) [inline docs/variable.properties]: 14.0.1?

Comment (paulbalaji) [reply docs/variable.properties]: there's no doc versions for patch releases

Comment (jamiebrynes7) [reply docs/variable.properties]: Wot he said ^

**Pull request merged**